ctapipe/image/cleaning.py

```diff
@@ -111,6 +113,74 @@ def tailcuts_clean(
         )
 
 
+def time_clustering(
+    geom,
+    image,
+    time,
+    minpts=5,
+    eps=1.0,
+    time_scale_ns=4.0,
+    space_scale_m=0.25,
+    hard_cut_pe=4,
+):
+    """
+
+    Clean an image by selecting pixels which pass a time clustering algorithm using DBSCAN.
+    Previously used for HESS [timecleaning]_.
+
+    As a neighbor-based image extractor algorithm can lead to biases in the time reconstruction of noise pixels,
+    specially those next to the shower, a cut in the minimum signal image with respect to the noise level is
+    firstly applied. The cut is performed relative to the noise to account for, e.g., bright stars. Alternatively,
+    a hard cut could also be performed.
+
+    DBSCAN runs with the reconstructed times and pixel positions after scaling. Scaling is needed because eps
+    is not dimension dependent. If scaling is performed properly, eps can be set to 1. DBSCAN returns the
+    cluster IDs of each point being -1 the label for noise pixels.
+
+    Parameters
+    ----------
+    geom: `ctapipe.instrument.CameraGeometry`
+        Camera geometry information
+    image: array
+        pixel charge information
+    time: array
+        pixel timing information
+    hard_cut_pe: float
+        Hard cut in the number of signal pe
+    minpts: int
+        Minimum number of points to consider a cluster
+    eps: float
+        Minimum distance in dbscan
+    time_scale_ns: float
+        Time scale in ns
+    space_scale_m: float
+        Space scale in m
+
+    Returns
+    -------
+    A boolean mask of *clean* pixels.
+    """
+    precut_mask = image > hard_cut_pe
+
+    arr = np.zeros(len(image), dtype=float)
+    arr[~precut_mask] = -1
+
+    pix_x = geom.pix_x.value[precut_mask] / space_scale_m
```

Comment:
space_scale_m seems to imply that `CameraGeometry` is in `CameraFrame`, however we are moving away from that.

Let's either call it just `space_scale` or actually use the camera geometry / camera readout information to define sensible values regardless of the frame.